subiquity/core.py

```diff
@@ -98,5 +111,12 @@ def unhandled_input(self, key):
         if key == 'f1':
             if not self.showing_help:
                 self.ui.right_icon.open_pop_up()
+        elif key == 'ctrl z':
```

Comment:
`ctrl z` is undo in s390x vt220 ascii terminal (because it's running in a web-browser). Can we have either different or an alternative key binding for this?

Ie. in grub/efi everything is ctrl+ and Fn key. Can we possibly change this to in `('ctrl z', 'f2')` ?

Reply:
yes, adding f2 as well makes sense.